test/shape_test.cpp

```diff
@@ -737,6 +738,22 @@ TEST_CASE(tuple)
     EXPECT(test::throws([&] { s.visit_type([](auto) {}); }));
 }
 
+TEST_CASE(return_shape_tuple)
+{
+    using migraphx::shape;
+    auto op = migraphx::make_op("@return");
+    std::vector<shape> s{{shape::float_type}, {shape::float_type}};
```

Comment:
You can also pass in You can also pass in `lens` and `strides` here while constructing the shape. It is optional though. It would just use default if they are not passed. But in practice it is better to test with lens and strides.

Reply:
I think while comparing the shape types, we are just doing a default constructor here, and checking again with defaults.. so will take the optional (easy) way out.. and leave it as it is..; Thanks.